### DISCUSSION TITLE
TIPS: Add Highcharts meteogram working with translations and dark mode

### DISCUSSION THREAD

**Discussion by norecords** [poblabs/weewx-belchertown#798] [Show and tell]
Hi everyone, I started a gist on January to implement a meteogram working with Highcharts, Belchertown skin translation and dark mode, it's using yr.no forecast api.
![Screenshot 2022-07-14 at 15-35-05 Météo Correns - Station météorologique en temps réel](https://user-images.githubusercontent.com/12499787/178995184-9ef0afd8-94da-47e5-a5ea-cd4b31296d8e.png)
Take a look to the source code -> [Meteogram for Belchertown skin](https://gist.github.com/norecords/85ea0186497404618574a7c62202f304)
See it in action -> [Météo Correns](https://meteo.correns.org/)

**Comment by CaptainSteubing** [comment]
Very nice and great work, thanks. Is there an easy way to convert the units from metric to imperial?

**Reply by norecords** [reply]
@CaptainSteubing After some tries you should convert data on this section
```
// Populate the parallel arrays
...
    y: node.data.instant.details.air_temperature,
...
```
to convert data from Celsius to Fahrenheit
` y: Math.round((node.data.instant.details.air_temperature*9/5) + 32), `

I'll work on it to be done from Weewx units
